website/docs/r/iam_access_key.html.markdown

```diff
@@ -55,6 +55,7 @@ The following arguments are supported:
 * `user` - (Required) The IAM user to associate with this access key.
 * `pgp_key` - (Optional) Either a base-64 encoded PGP public key, or a
   keybase username in the form `keybase:some_person_that_exists`.
+* `status` - (Optional) The access key status to apply. Defaults to "Active".
```

Comment:
Can you please note here that the other valid value is `Inactive`. Thanks!

Reply:
Added a line with the set of valid values.